test/features/support/fixtures.js

```diff
@@ -6,5 +6,8 @@ module.exports = {
     sourceUrl: 'http://example.com',
     rating: '4',
     deadline: '20/10/2016',
+    deadlineDate: null,
+    applicationDate: null,
+    interviewDate: null,
```

Comment:
Have changed this to be more relevant for the status selected e.g. added status = applied and a applicationDate.  Removed the nulls.